apps/projects/helpers.py

```diff
@@ -41,3 +43,12 @@ def get_num_reported_unread_comments(project):
         .filter(num_reports__gt=0)
         .count()
     )
+
+
+def get_pagination(obj: object, project: object, per_page=15) -> int:
+    objects = obj.__class__.objects.filter(module=project)
```

Comment:
rename parameter `project` to `module` in function `get_pagination`. Either it is a project or a module, but `filter(module=project)` looks wrong.